src/cmd/cmd.h

```diff
@@ -22,13 +22,16 @@
  * @brief Helper class for command line client
  * @ingroup cmd
  */
-class Cmd : public QObject {
+class Cmd : public QObject
+{
     Q_OBJECT
 public:
-    Cmd() : QObject() { }
-public slots:
-    void transmissionProgressSlot() {
+    Cmd()
+        : QObject()
+    {
     }
```

Comment:
I don't think there's a `AllowCtorSingleLine`, so probably it's just the generic formatting for these.